src/components/Forms/MaskedTextField.js

```diff
@@ -285,6 +290,7 @@ class MaskedTextField extends React.Component {
             name={name}
             aria-required={required}
             aria-invalid={hasError}
+            aria-describedby={hasError ? `hint_${inputId} error_${inputId}` : `hint_${inputId}`}
```

Comment:
Does a screenreader read the text from both of these elements if there's an error?

Reply:
Actually just learned about this on Monday at workshop. Yeah, aria-labelledby and aria-describedby accept multiple ids just like class names. Pretty cool.

Reply:
Correct, it'll read off in order of each element listed